pkg/openstack/loadbalancer_test.go

```diff
@@ -534,3 +536,30 @@ func Test_getListenerProtocol(t *testing.T) {
 		})
 	}
 }
+
+func Test_getSecurityGroupName(t *testing.T) {
+
+	service := &corev1.Service{ObjectMeta: v1.ObjectMeta{UID: "12345", Namespace: "security-group-namespace", Name: "security-group-name"}}
+	expected := "lb-sg-12345-security-group-namespace-security-group-name"
+	t.Run("regular test security group name and length", func(t *testing.T) {
+		if got := getSecurityGroupName(service); got != expected {
+			t.Errorf("getSecurityGroupName() = %s expected = %s", got, expected)
+		}
+	})
+
+	service = &corev1.Service{ObjectMeta: v1.ObjectMeta{UID: "12345678-90ab-cdef-0123-456789abcdef", Namespace: "security-group-longer-test-namespace", Name: "security-group-longer-test-service-name-with-more-than-255-byte-this-test-should-be-longer-than-255-i-need-that-ijiojohoo-afhwefkbfk-jwebfwbifwbewifobiu-efbiobfoiqwebi-the-end-e-end-pardon-the-long-string-i-really-apologize-if-this-is-a-bad-thing-to-do"}}
+	expected = "lb-sg-12345678-90ab-cdef-0123-456789abcdef-security-group-longer-test-namespace-security-group-longer-test-service-name-with-more-than-255-byte-this-test-should-be-longer-than-255-i-need-that-ijiojohoo-afhwefkbfk-jwebfwbifwbewifobiu-efbiobfoiqwebi-the-end"
```

Comment:
Let's make sure the tests are readable with proper lining & spacing in the object declarations.